src/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -149,7 +149,7 @@ private HttpRequestMessage CreateRequestMessage(Uri endpoint, IDictionary<string
             }
             catch (TaskCanceledException exception)
             {
-                requestContext.Logger.ErrorPii(exception);
+                requestContext.Logger.Error("The HTTP request failed or it was cancelled. " + exception.Message);
```

Comment:
>cancelled [](start = 79, length = 9)

I believe American English this would be: canceled ?

Reply:
Oh, you are breaking my bones here Henrik. Ok ...